Resources/Textures/Interface/Actions/actions_borg.rsi/meta.json

```
@@ -1,7 +1,7 @@
 {
     "version": 1,
     "license": "CC-BY-SA-3.0",
-    "copyright": "Taken from vgstation at commit https://github.com/vgstation-coders/vgstation13/commit/cdbcb1e858b11f083994a7a269ed67ef5b452ce9",
+    "copyright": "Taken from vgstation at commit https://github.com/vgstation-coders/vgstation13/commit/cdbcb1e858b11f083994a7a269ed67ef5b452ce9, Module actions by Scarky0 at https://github.com/space-wizards/space-station-14/pull/32505. chem, adv-chem, and adv-mining by mubururu_",
```

You don't need to reference the PR that adds them since Scar is the one that made them originally, he didn't iterate or take from another codebase/repo.

```suggestion
    "copyright": "Taken from vgstation at commit https://github.com/vgstation-coders/vgstation13/commit/cdbcb1e858b11f083994a7a269ed67ef5b452ce9, Module actions by Scarky0. chem, adv-chem, and adv-mining by mubururu_",
```